creator/data_reviews/signals.py

```diff
@@ -0,0 +1,64 @@
+from django.dispatch import receiver
+from django_fsm.signals import post_transition
+from django.db.models.signals import pre_delete
+
+from creator.files.models import File
+from creator.events.models import Event
+from creator.data_reviews.models import DataReview, State
+
+
+@receiver(post_transition, sender=DataReview)
+def data_review_post_transition(
+    sender, instance, name, source, target, *args, **kwargs
+):
+    """
+    Create a corresponding Event after the state of a DataReview is changed.
+    """
+    event = None
+    if (source == State.NOT_STARTED) and (target == State.IN_REVIEW):
```

Comment:
Does it make sense to do the event creation inside the model's transitions instead of in a signal?

Reply:
Hmm, possibly. I guess I can't think of a good reason why the event should be fired _after_ the transition occurs. Although I will have a circular import, since Event imports DataReview. Maybe I can avoid that by putting the Event import inside each transition method?